### ISSUE TITLE
Registration/Login Page Redesign

### ISSUE THREAD

**Issue by jeffreyxchan** [ideahacks/ideahacks.la#81]
**Summary**
IT'S TIME.
Of all the designs that Allison has recently sent us, I think the registration/login page is the least likely to change over time, so I think it's safe to start building it. So I'm going to put you on rebuilding both of them to look like her designs.

I would recommend starting with making the registration page, and then making the login page will just be a lot of copy and paste.

**Important Notes**
1. Check the website slack for the pdf of of allison's designs and refer to the login/registration pages. They should be the pages near the top.

2. Checkout the style guide on the first page. It'll list out the fonts and colors that you should be using. If you're missing any images/logos/pictures, please feel free to hit up allison kim on our slack.

3. Good CSS properties you should use when making the design:
`max-width` - puts a cap on the width of an element, but doesn't restrict how much can shrink. Will probably be used to restrict the width of the main box that surrounds the registration form
```
position: absolute;
top: 50%;
left: 50%;
transform: translate(-50%, -50%);
```
^^^^ a really solid combo for centering elements within a parent div. Will probably be used to center the main box that surrounds the form. You just need to make sure that the parent element (probably the `html` and `body` tags) have a height and width of 100% for it to work.

4. Don't worry about writing bad html or css. I'll review your code with extra scrutiny and pick out everything that needs to be changed, or need to be coded with better style.

5. Write CSS that is specific for your registration page. Your CSS should be specific enough that it will only work on the registration and login pages, and no other page. You'll probably be creating one CSS file called `registration.css` and using that CSS for both the login and registration pages.

6. Also, use your best judgement on what sort of CSS will be global. For example, fonts, font colors, font sizes are all properties that will be shared among all pages on our site, so those stylings should go within `style.css` which is a global stylesheet that'll be included on all pages.

The styling of the registration form elements are also similar to the application styling. For example the input fields are styled the same way so that they only have a bottom border and no internal padding. Thus it makes sense to also separate out CSS of form elements into another file called form.css, a global stylesheet that will style all of our form elements.

So in summary, when writing CSS, please use your best judgement to separate it into three groups:
style.css - a super global stylesheet that'll be used for all pages
form.css - a global stylesheet that'll be used for any page containing form elements
registration.css - a sheet that'll only be used for your registration/login pages, and nothing else.
